hybrid/flow.py

```diff
@@ -163,6 +172,104 @@ def halt(self):
             component.stop()
 
 
+class Branches(Runnable, traits.MIMO):
+    """Parallelly executed :class:`~hybrid.core.Runnable` components.
```

Comment:
Grammatically correct to be sure but it is unidiomatic. ":class:`~hybrid.core.Runnable` components executed in parallel" would be more standard. For your consideration

Reply:
Can you add something about the difference between this component and RacingBranches? Maybe just point out that branches run till all terminate rather than the first termination?